index.html

```diff
@@ -27,7 +27,9 @@
 <body>
 	<header>
 		<h1>Learn Teach Code</h1>
-		<h5><a href="https://learnteachcode.org/slack" target="_blank" rel="noopener">Join our Slack!</a> | <a href="https://learnteachcode.org/code-coffee-compendium" target="_blank" rel="noopener">Resources</a> | <a href="./code-of-conduct.html">Code of Conduct</a></h5>
+		<h5><a href="https://learnteachcode.org/slack" target="_blank" rel="noopener">Join our Slack!</a> | <p>
+			<strong><a href="https://discord.gg/zSEyPCsFWD" target="_blank" rel="noopener">Join our Discord!</a></strong>
+		</p> | <a href="https://learnteachcode.org/code-coffee-compendium" target="_blank" rel="noopener">Resources</a> | <a href="./code-of-conduct.html">Code of Conduct</a></h5>
```

Comment:
I think it's invalid HTML to have a <p> tag in an heading tag so let's just remove that extra tag